File: app/src/main/java/closer/vlllage/com/closer/store/models/GroupAction.java
package closer.vlllage.com.closer.store.models;

import io.objectbox.annotation.Entity;

@Entity
public class GroupAction extends BaseObject {
    private String group;
    private String name;

    public String getGroup() {
        return group;
    }

    public GroupAction setGroup(String group) {
        this.group = group;
        return this;
    }

    public String getName() {
        return name;
    }

    public GroupAction setName(String name) {
        this.name = name;
        return this;
    }
}

File: app/src/main/java/closer/vlllage/com/closer/store/models/GroupAction.kt
package closer.vlllage.com.closer.store.models

import io.objectbox.annotation.Entity
import java.util.*

@Entity
class GroupAction : BaseObject() {
    var group: String? = null
    var name: String? = null
    var intent: String? = null
    var about: String? = null
    var photo: String? = null
    var flow: String? = null
    var used: Date? = null
}
